README.md

```diff
@@ -1,2 +1,165 @@
-# kss-caxy-zaba-template
-Caxy's Zaba kss-node builder template.
+# Caxy's Zaba Theme KSS Builder
+
+TODO: How to Generate Styleguide and Viewing the Styleguide sections
+likely need revision.
+
+This styleguide setup and builder has the following features:
+
+- Color Swatches
+- Pattern Markers
+
+
+## How to Generate Styleguide
+
+**From within the /src/styleguide folder**
+
+Run `npm install` to get up and running. You can then run the following
+to build your styleguide:
+
+`npm run generate-styleguide`
+
+This will compile the styleguide's CSS and create the styleguide pages 
+based on comments found in files in the `/src/styles/` folder and the
+markup HTML found in the `src/styleguide/pattern-markup/` folder.
+
+
+## Viewing the Styleguide
+
+While in the src/styleguide/ folder, running `npm start` will start a
+local server to aid in review, which is recommended. This will get
+around quirks that can crop up with font rendering outside of a server
+setting.
+
+
+## Populating and Editing the Styleguide
```

Comment:
One thing I realized when I first started looking at the kss-node stuff and the template was that I actually have no idea what the custom addons for kss-node do ... I wonder if it would be valuable to list the custom addons we use for this template and what each of them do?

for context I'm referring to the config like:

```
[
     'hidemarkup',
     'patterntype',
     'containspatterns',
     'devnotes',
     'status'
   ],
```